portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/IndividualsEnrichedProfiles.testcase

```diff
@@ -112,15 +112,27 @@ definition {
 				count = "0 Profiles");
 		}
 
-		task ("Sync the new user and view the blog") {
+		task ("Sync the new user") {
 			ACUtils.launchDXP();
 
 			ACDXPSettings.syncNewContactData();
+		}
+
+		task ("Sign out") {
+			UserBar.signOut();
+		}
 
-			User.logoutAndLoginPG(
+		task ("Clear local storage") {
+			ACUtils.clearLocalStorage();
+		}
```

Comment:
Hey @joshitagaki, I think maybe this change to clear the local storage is not necessary in this case, as before you are logging in with user **AC AC**, you were logging in with TEST TEST, so when you change the user the userId will be modified, so it would not be necessary to clear the local storage. I think that's why the test is passing the release routine. What do you think?

Reply:
Oh yea you're right plus it looks like the contacts weren't synced when TEST TEST was logged in so it would be anonymous. Guess it failed before due to flaky behavior

Reply:
I'll close out this PR and the ticket since I didn't find any tests that are particularly affected by the changes